styles.css

```diff
@@ -92,3 +100,161 @@ html {
 .social-icon {
   height: 24px;
 }
+
+/* SECTION WORK */
+
+.work {
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  background-color: #fff;
+  width: 90%;
+  height: 40rem;
+  border-radius: 9px;
+  margin-bottom: 100px;
+}
+
+.content {
+  display: grid;
+  grid-template-rows: 1fr 2fr;
+  padding-top: 20px;
+}
+
+.snapshoot {
+  margin-left: 22px;
+}
+
+.tonic {
+  padding: 10px 0 5px 20px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.detail {
+  padding: 0 20px 0 20px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.profile {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start). 

- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.Flex is just an example you can do it to duplicate properties.



- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)